submerge:
  defaults:
    submerge_depth: -150
    timeout_time: 10


spin:
  defaults:
    fallback_heading: 45


gotobuoy:
  defaults:
    forward_speed: 1600
    heading_thld: 20
    timeout: 15
    zed_fov: 110 #: Not sure what exact degree is, 110 for now